package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "backbone": "~1.3.2",
     "coffee-script": "1.6.1",
-    "edx-pattern-library": "~0.12.4",
+    "edx-pattern-library": "git+https://github.com/edx/ux-pattern-library.git#f8d8dbcb86c22450566dc6b40db0f23ea0c6fcba",
```

Comment:
I was planning on waiting for a Pattern Library release. IMO we shouldn't have code on master that refers to particular commits.